frontend/app/forms/inputs/select.component.js

```diff
@@ -5,15 +5,16 @@ import { DataContainerContext } from "../data-container.component";
 
 export default function Select(props) {
   const dataContext = React.useContext(DataContainerContext);
-  const [value, setValue] = React.useState(
-    get(dataContext.data, props.dataKey)
```

Comment:
although it was looking in context, if the first render happened before the data loaded, you'd get undefined in your state